### ISSUE TITLE
Appending classes to svg doesn't work when the svg already has a class attribute in a nested node

### ISSUE THREAD

**Issue by mhanberg** [nikkomiu/phoenix_inline_svg#10]
## Description

When attempting to apply a class attribute to an svg that has child node which have class attributes, the function will append your new class to those child attributes and not create a new class attribute on the root `svg` node.

```elixir
svg_image(@conn, "my-svg", class: "h-16")
```

```svg
<svg>
  <path class="class1"></path>
  <path class="class2"></path>
</svg>
```

## Expected

```svg
<svg class="h-16">
  <path class="class1"></path>
  <path class="class1"></path>
</svg>
```

## Actual

```svg
<svg>
  <path class="class1 h-16"></path>
  <path class="class1 h-16"></path>
</svg>
```

## Work around

I hard coded the fill values for the child nodes that were uses classes.

**Comment by gillisig** [comment]
Can confirm this bug, same thing happens for me.

**Comment by nikkomiu** [comment]
It kind of makes sense based on the implementation of that caching mechanism looking at it. Obviously not the expected behavior. I think that a parsing library will likely have to be introduced to dynamically add attributes to the post-cached versions.

If either of you want to try to get a PR in for the fix that would be great! I won't have time to get to it for at least a few weeks.

**Comment by mhanberg** [comment]
@nikkomiu I'm going to take a crack at this

**Comment by nikkomiu** [comment]
(May be somewhat pointless since you wrote the changes but) PR #13 is merged to master. @mhanberg do you want to test against the master branch of the repo?

**Comment by mhanberg** [comment]
I can later this weekend.

**Comment by mhanberg** [comment]
Confirmed the fix.

**Comment by nikkomiu** [comment]
Released under v1.3.0
